doc/source/examples/dem/silo/silo.rst

```diff
@@ -0,0 +1,197 @@
+==================================
+Silo
+==================================
+
+This is the sixth example of Lethe-DEM. This example simulates the filling and discharge of particles in a wedge-shaped silo. We set up this simulation according to the experiments of Golshan et al. [1]. It is recommended to visit `DEM parameters <https://lethe-cfd.github.io/lethe/parameters/dem/dem.html>`_ for more detailed information on the concepts and physical meanings of the parameters in Lethe-DEM.
+
+Features
+----------------------------------
+- Solvers: ``dem_3d``
+- Floating walls
+- Gmsh grids
+- Checkpointing (restart)
+
+
+Location of the examples
+------------------------
+ ``/examples/dem/3d_silo/silo_Golshan.prm``
+
+
+Description of the case
+-----------------------
+
+This simulation consists of two stages: filling (0-4 s) and discharge (4-40 s) of particles. During the filling stage, we use a stopper (floating wall) to keep the inserted particles in the hopper region of the silo. When all the particles are inserted and packed in the hopper, we remove the stopper and particles leave the hopper.
+
+
+Parameter file
+--------------
+
+Mesh
+~~~~~
+
+Despite previous examples, in this example, we use a mesh generated using Gmsh. The Gmsh (with an extension of .msh) file is located inside the example folder.
```

Comment:
```suggestion
Contrary to previous examples, in this example, we use a mesh generated using Gmsh. The Gmsh (with an extension of .msh) file is located inside the example folder.
```

Reply:
I don't think the word despite can be used in this context.

Reply:
Corrected.